react/FilterNavigatorFlexible.js

```diff
@@ -7,107 +7,111 @@ import FilterNavigatorContext from './components/FilterNavigatorContext'
 import styles from './searchResult.css'
 import { sortFilterValues } from './utils/sortFilterValues'
 
-const withSearchPageContextProps = (Component) => ({
-  layout,
-  initiallyCollapsed,
-  scrollToTop,
-  maxItemsDepartment,
-  maxItemsCategory,
-  categoryFiltersMode,
-  filtersTitleHtmlTag,
-  truncateFilters,
-  openFiltersMode,
-  closeOnOutsideClick,
-  appliedFiltersOverview,
-  totalProductsOnMobile,
-  fullWidthOnMobile,
-  navigationTypeOnMobile,
-  updateOnFilterSelectionOnMobile,
-  showClearByFilter,
-  priceRangeLayout,
-  facetOrdering = [],
-}) => {
-  const {
-    searchQuery,
-    map,
-    params,
-    priceRange,
-    hiddenFacets,
-    filters,
-    showFacets,
-    preventRouteChange,
-    facetsLoading,
-  } = useSearchPage()
+const withSearchPageContextProps =
```

Comment:
Can you lint this file?

Reply:
Yes. I had a problem with the configuration. I fixed all errors in the files i made changes in.